test/rules/no-shadowed-variable/temporal-dead-zone/test.ts.lint

```diff
@@ -0,0 +1,89 @@
+['a', 'b', 'c'].map((path) => path.resolve(path))
+                     ~~~~ [err % ('path')]
+
+import * as path from 'path'
+
+export function foo(wam: boolean) {
+    if (wam) {
+        const now = new Date();
```

Comment:
would you mind marking this up with comments to indicate where the option is taking effect? similar to your example in the docs, just a simple `// not an error with tdz disabled`